src/index.js

```diff
@@ -21,6 +22,15 @@ const publicDirPath = path.join(__dirname, '../public')
 
 app.use(express.static(publicDirPath))
 
+app.get('/rooms', async (req, res) => {
+    try {
+        const rooms = getRooms()
+        res.status(200).send(rooms)
+    } catch (error) {
+        res.status(500).send(error)
```

Comment:
in any error case it sends empty json object in to clients

Reply:
Changed the `.status(500)` line to this: `res.status(500).send({ error: error.message })`.

Added this to the getRooms method to 'emulate' an error:
`if (uniqueRooms.length === 0) {
        throw new Error ('No existing rooms')
    }
    return uniqueRooms`

Is this ok?